SU-HSDO/suhumsci
# Arch 293 js menu fix

# READY FOR REVIEW

# Summary
- Adds js function to determine if submenu will be too close to the right edge, and reposition it if so.

# Steps to Test
1. Watch [this video](https://drive.google.com/file/d/1TK3fNaxfOWFwcpDFYD7egm3T94EAa51U/view?usp=sharing)
1. Explore the main menu and its dropdowns at various breakpoints
1. Confirm that submenu appears at the right spot at all times

# Associated Issues and/or People
- [JIRA ticket](https://stanfordits.atlassian.net/browse/ARCH-311)
